### ISSUE TITLE
License

### ISSUE THREAD

**Issue by fakerybakery** [declare-lab/tango#40]
Hi, thanks for making this great model. On your webpage, you claim that this model is "open source," however the CC-BY-NC-ND license is not open source. [According to the OSI](https://opensource.org/osd/), open source software must not restrict commercial use:

> 6. No Discrimination Against Fields of Endeavor
> The license must not restrict anyone from making use of the program in a specific field of endeavor. For example, it may not restrict the program from **being used in a business**, or from being used for genetic research.

I would deeply appreciate it if you could make this model truly open source. Thank you!

**Comment by fakerybakery** [comment]
Hi, congratulations on the Tango 2 release! I was wondering if there was any update on this? Thanks!

**Comment by soujanyaporia** [comment]
Hi, this comes with a non-commercial license. Thank you!

**Comment by fakerybakery** [comment]
Hi, thanks for your response! I noticed on your webpage you said this code is "open sourced," however NC licenses do not comply with the OSI's OSD, which means they are not open source. Are there any plans to switch to an open source license?
Thanks for releasing this great project!

**Comment by soujanyaporia** [comment]
Yes, understood. I think we need to discuss about the commercial use. If you are keen, please reach out to me by email: sporia@sutd.edu.sg. Thank you!

**Comment by fakerybakery** [comment]
Thanks! I wasn’t going to use it for commercial purposes, but was interested in potentially integrating it into an open source project and allow commercial use of that open source project to others.
